### PR TITLE
xcorr fix

### PR DESCRIPTION
**Pull request by trishorts**

### CODE REVIEW

File: MassSpectrometry/MsDataFile.cs [hunk 1]
```diff
@@ -173,39 +173,52 @@ public static void WindowModeHelper(ref double[] intensities, ref double[] mArra
...
+                }
+            }
+
             List<double> reducedMzList = new List<double>();
             List<double> reducedIntensityList = new List<double>();
 
             foreach (int rangeIndex in mzInRange.Keys)
             {
-                List<double> tempIntList = new List<double>();
+                //List<double> tempIntList = new List<double>();
```
Comment (zrolfs) [inline MassSpectrometry/MsDataFile.cs]: delete commented code

File: MassSpectrometry/MzSpectra/MzSpectrum.cs [hunk 2]
```diff
@@ -401,8 +400,14 @@ public void XCorrPrePreprocessing(double scanRangeMinMz, double scanRangeMaxMz,
                         denominator--;
                     }
                 }
-                scaledIntensities[i] = Math.Max(0, genericIntensityArray[i] - 1d / (((double)Math.Max(1,denominator)) * scaleValue));
+                if(i>200 && genericIntensityArray[i] == 0)
```
Comment (zrolfs) [inline MassSpectrometry/MzSpectra/MzSpectrum.cs]: remove debug code

File: MassSpectrometry/MsDataFile.cs (hunk 1 above)
Comment (trishorts) [reply MassSpectrometry/MsDataFile.cs]: done

File: MassSpectrometry/MzSpectra/MzSpectrum.cs (hunk 2 above)
Comment (trishorts) [reply MassSpectrometry/MzSpectra/MzSpectrum.cs]: done

**Pull request merged**